src/CatalogService/src/Domain/Common/BaseEntity.cs

```diff
@@ -1,26 +1,32 @@
 ﻿using System.ComponentModel.DataAnnotations.Schema;
+using HotChocolate;
 
 namespace CatalogService.Domain.Common;
 
 public abstract class BaseEntity
 {
     public int Id { get; set; }
 
+    [GraphQLIgnore]
     private readonly List<BaseEvent> _domainEvents = new();
 
     [NotMapped]
```

Comment:
Domain should not know about infrastructure

Reply:
done